### PR TITLE
Remove unused parameter in FeedbackResponseCommentsLogic.java #6639

### LINKED ISSUE #6639: Remove unused parameter in FeedbackResponseCommentsLogic.java
In `teammates.logic.core.FeedbackResponseCommentsLogic#isResponseCommentVisibleForUser`, there are three unused parameters: `courseId`, `section`, `instructor`. They all can be removed.

### PR DESCRIPTION
**Pull request by smithla1**
Fixes #6639 
Remove unused parameter in FeedbackResponseCommentsLogic.java #6639 

I removed three parameters (courseID, section, and instructor) from
teammates.logic.core.FeedbackResponseCommentsLogic#isResponseCommentVisibleForUser.
These parameters were no longer used in the method.

In addition, I removed the parameters courseID, section, and instructor from calls to
the method teammates.logic.core.FeedbackResponseCommentsLogic#isResponseCommentVisibleForUser.

This pull request is ready to be reviewed.

### CODE REVIEW

**Comment by ablyx** [comment]
Hi @smithla1, welcome and good job! Your PR is currently failing a small check. You can click on the red cross to find out why. 

Hint: `:pmdMain
/home/travis/build/TEAMMATES/teammates/src/main/java/teammates/logic/core/FeedbackSessionsLogic.java:1712:	Avoid unused local variables such as 'instructor'.`

File: src/main/java/teammates/logic/core/FeedbackResponseCommentsLogic.java [hunk 1]
```diff
@@ -286,10 +286,10 @@ private boolean isFeedbackParticipantNameVisibleToUser(FeedbackResponseAttribute
      * Verify whether the comment is visible to certain user
      * @return true/false
      */
-    public boolean isResponseCommentVisibleForUser(String userEmail, String courseId, UserRole role,
-            String section, StudentAttributes student, Set<String> studentsEmailInTeam,
+    public boolean isResponseCommentVisibleForUser(String userEmail, UserRole role,
+            StudentAttributes student, Set<String> studentsEmailInTeam,
             FeedbackResponseAttributes response, FeedbackQuestionAttributes relatedQuestion,
-            FeedbackResponseCommentAttributes relatedComment, InstructorAttributes instructor) {
+            FeedbackResponseCommentAttributes relatedComment) {
```
Comment (samsontmr) [inline src/main/java/teammates/logic/core/FeedbackResponseCommentsLogic.java]: Try to use as few lines as possible for the method signature

File: src/main/java/teammates/logic/core/FeedbackSessionsLogic.java [hunk 2]
```diff
@@ -1720,9 +1720,9 @@ private FeedbackSessionResultsBundle getFeedbackSessionResultsForUserInSectionBy
         for (FeedbackResponseCommentAttributes frc : allResponseComments) {
             FeedbackResponseAttributes relatedResponse = relevantResponse.get(frc.feedbackResponseId);
             FeedbackQuestionAttributes relatedQuestion = relevantQuestions.get(frc.feedbackQuestionId);
-            boolean isVisibleResponseComment = frcLogic.isResponseCommentVisibleForUser(userEmail, courseId,
-                    role, section, student, studentsEmailInTeam, relatedResponse,
-                    relatedQuestion, frc, instructor);
+            boolean isVisibleResponseComment = frcLogic.isResponseCommentVisibleForUser(userEmail,
+                    role, student, studentsEmailInTeam, relatedResponse,
+                    relatedQuestion, frc);
```
Comment (samsontmr) [inline src/main/java/teammates/logic/core/FeedbackSessionsLogic.java]: Same comment as above

File: src/main/java/teammates/logic/core/FeedbackSessionsLogic.java [hunk 3]
```diff
@@ -1953,9 +1953,9 @@ private FeedbackSessionResultsBundle getFeedbackSessionResultsForUserWithParams(
             for (FeedbackResponseCommentAttributes frc : allResponseComments) {
                 FeedbackResponseAttributes relatedResponse = relevantResponse.get(frc.feedbackResponseId);
                 FeedbackQuestionAttributes relatedQuestion = relevantQuestions.get(frc.feedbackQuestionId);
-                boolean isVisibleResponseComment = frcLogic.isResponseCommentVisibleForUser(userEmail, courseId,
-                        role, section, student, studentsEmailInTeam, relatedResponse,
-                        relatedQuestion, frc, instructor);
+                boolean isVisibleResponseComment = frcLogic.isResponseCommentVisibleForUser(userEmail,
+                        role, student, studentsEmailInTeam, relatedResponse,
+                        relatedQuestion, frc);
```
Comment (samsontmr) [inline src/main/java/teammates/logic/core/FeedbackSessionsLogic.java]: as above.

File: src/main/java/teammates/logic/core/FeedbackResponseCommentsLogic.java (hunk 1 above)
Comment (smithla1) [inline src/main/java/teammates/logic/core/FeedbackResponseCommentsLogic.java]: Thank you for the suggestions!

**Comment by smithla1** [comment]
@ablyx, thank you for your assistance! I have updated my pull request accordingly.

**Pull request merged**